yamale/yamale.py

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env python
 import sys
+import os
 from .schema import Schema
 
 PY2 = sys.version_info[0] == 2
 
 
 def make_schema(path, parser='PyYAML', validators=None):
     # validators = None means use default.
+    # test if the schema.yaml file is empty
+    try:
+        if not os.path.getsize(path):
+            raise ValueError('schema.yaml is an empty file!')
+    except ValueError as e:
+        print(e)
```

Comment:
Since we're in the API layer, we shouldn't be printing anything. All you need to do is raise the error.

Reply:
Good item.